src/app/time-travel-intro/collaborative-devtools/page.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Collabritve devtools
+---
+
+The best part of software development is that it's a team sport that allows you to work on your own schedule. It's so good that we take it for granted, and forget that there was ever a time when you couldn't open a PR and ask for a review.
+
+One of our goals is to bring this workflow to debugging. The reason why it's difficult to collaborate on a difficult problem today is that for someone to help you, they first have to see the problem you're looking at which means stopping what they're doing and getting on a Zoom call or trying to reproduce the problem you're investigating.
+
+Replay makes it possible to asynchronously collaborate on a bug because when you're viewing a replay, there's a browser deterministically replaying the bug in the cloud. This means that when you're investigating a difficult problem inside of a Redux thunk, you can drop a comment, at mention a team member, and they can take a look when they're available.
+
+When they open the replay, they can jump to the same point in the execution you were looking at and inspect the local state as if they were pairing with you initially.
```

Comment:
This intro is written well, but feels a bit long. I think we should broadly work on shortening the contents of this headings collection.

I think we can ship this as-is (typo not withstanding) and do a follow-up PR to shorten it

Reply:
Nods. I'm on the fence about this time trave intro. On one hand, I like it if it's in a guides section and you want to read more. I also could see us tightening it up and keeping it here at the top.